imsv-docs-docusaurus/openapi/endpoints/login/logout.yaml

```diff
@@ -0,0 +1,21 @@
+post:
+  tags:
+    - authentication
+  summary: Logout
+  description: |
+    The logout endpoint invalidates  the current session to ensure it
+    can no longer be used for accessing protected resources within the Immersve API.
+    All access tokens connected with the session will become unusable.
+  responses:
+    "200":
+      description: Successful logout
+      content:
+        application/json:
+          schema:
+            $ref: "./models/logout-response.yaml"
+    "401":
+      description: Unauthorized or missing authentication credentials.
+      content:
+        application/json:
+          schema:
+            $ref: "../../models/api-error-401.yaml"
```

Comment:
Technically every endpoint can return a 401.
If we can add it in one place for all of them then great. If we need to add it to each endpoint, then I would do it as a separate PR and leave it out in this one.

Reply:
Removed